themes/default/content/blog/easy-langserve-apps-with-pulumi/index.md

```diff
@@ -0,0 +1,437 @@
+---
+title: "Easy LangServe Apps with Pulumi on AWS"
+date: 2024-02-13
+meta_desc: "Create a LangServe app. With Pulumi, you can create, deploy, and manage Langserve apps using your favorite language."
+meta_image: meta.png
+authors:
+- engin-diri
+
+tags:
+- ai
+- langserve
+- aws
+---
+
+![Easy LangServe Apps with Pulumi on AWS](./meta.png)
```

Comment:
For future, please don't add these images here like this. If we want to have these images appear at the top of a blog post, we can do it for all posts in the template.

Reply:
@cnunciato should I remove it?

Reply:
All good, I'll fix it up later -- going to add a timestamp to the post in a moment to make sure it sorts correctly once Luke's post goes out.